.travis.yml

```diff
@@ -10,6 +10,8 @@ bundler_args: --without production
 before_script:
   - mkdir -p tmp/Dataload/EndowRpt
   - mkdir -p tmp/Dataload/BookplateMerge/Batches/Queue
+  - sudo mkdir -p /symphony/Dataload/UserloadRerun
+  - sudo mkdir -p /symphony/Dataload/ILLiadUserExport
```

Comment:
Can these be added to tmp/Dataload instead of /symphony/Dataload? I find tests that require files in these directories fail when running them locally.